.github/ISSUE_TEMPLATE/development_task.yaml

```diff
@@ -0,0 +1,19 @@
+name: "⚙️ Development task"
+description: Track and plan development work
+labels: kind:dev-task
+body:
+  - type: textarea
+    id: abilities
+    attributes:
+      label: Description
+      value: What's this task about?
+    validations:
+      required: true
+
```

Comment:
Could we request a "Why ?" type of field here ? Something explaining why the task is necessary (different from the description)

Reply:
I'd rather have one field for both description and goal: according to my experience with my previous template, describing the task often results in describing the goal

Reply:
I suggest
```
     label: Description and goal
     value: What's this task about, and what is it for?
```